docs_user/modules/proc_stopping-infrastructure-management-and-compute-services.adoc

```diff
@@ -28,19 +26,23 @@ computes=(
 ----
 +
 ** Replace `["standalone.localdomain"]="192.168.122.100"` with the name of the Compute node and its IP address.
-** These ssh variables with the ssh commands are used instead of ansible to create instructions that are independent of where they are running. But ansible commands could be used to achieve the same result if you are in the right host, for example to stop a service:
+** Replace `<path_to_SSH_key>` with the path to your SSH key.
 +
+[NOTE]
+====
+These ssh variables with the ssh commands are used instead of ansible to create instructions that are independent of where they are running. But ansible commands could be used to achieve the same result if you are in the right host, for example to stop a service, run the following command:
```

Comment:
Can we specify what the “right host” is?

Reply:
normally we should name it somthing like and ansible control node

Reply:
This can be removed now as there is a special adoption.yaml task to stop nova compute related services, including libvirt daemons https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_nova/tasks/main.yml#L18 and https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_libvirt/tasks/adoption.yml

Reply:
Ok, I'll remove the note.